README.md

```diff
@@ -68,7 +68,35 @@ One can create a new header file by using the command: `javac -h . ED25519.java`
 
 The output will be a header file, the contents hereof must be matched appropriately as described in the [documentation](https://docs.rs/jni/0.19.0/jni/) into the [lib.rs](crypto-jni/src/lib.rs) rust source file.
 
+## Android
+To build the android AAR package, one has to do the following:
 
+1. Clone this repository and remember to update git submodules, for example when cloning via
```

Comment:
Are there any prerequisites for building the `.aar` file? E.g. `cargo-ndk` ?

Reply:
Yes :disappointed:
I've added a step to add cargo-ndk, and added more about the Android SDK requirement.